operator/data/profiles/default.yaml

```diff
@@ -356,7 +356,7 @@ spec:
     telemetry:
       enabled: true
```

Comment:
I tried this yesterday, seems like telemetry v1 (mixer) also be installed by this telemetry.enabled=true. Is this what we want to keep?

Reply:
Yes. We want v2 on, which will install the EnvoyFilters and set some flag like `disableHttpMetrics`. We stll have mixer for TCP telemetry (for now)

Reply:
got it, just a double check. 

LGTM.